libraries/mbed/targets/hal/TARGET_WIZNET/TARGET_W7500x/us_ticker.c

```diff
@@ -33,9 +33,12 @@
 #include "PeripheralNames.h"
 #include "system_W7500x.h"
 
+#define TIMER_0         DUALTIMER0_0
+#define TIMER_1         PWM_CH1
+#define TIMER_IRQn      DUALTIMER0_IRQn
 
-static PWM_TimerModeInitTypeDef TimMasterHandle_CH3;
-static PWM_TimerModeInitTypeDef TimMasterHandle_CH2;
+static PWM_TimerModeInitTypeDef TimerInitType;
+static DULATIMER_InitTypDef TimerHandler;
```

Comment:
Is this typo DUALTIMER?

Is PWM used for timekeeping and DUALTIMER for interrupts?